### PR TITLE
LPS-125128 prevent NPE when structure has a field group

### PR DESCRIPTION
**Pull request by marcosapmf**

### CODE REVIEW

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/java/com/liferay/dynamic/data/mapping/form/renderer/internal/DDMFormFieldTemplateContextFactory.java [hunk 1]
```diff
@@ -280,6 +280,10 @@ protected DDMFormFieldRenderingContext createDDDMFormFieldRenderingContext(
 
 		List<Map<String, String>> list = new ArrayList<>();
 
+		if (ddmFormFieldOptions == null) {
```
Comment (jeyvison) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/java/com/liferay/dynamic/data/mapping/form/renderer/internal/DDMFormFieldTemplateContextFactory.java:283]: Use ListUtil.isEmpty

**Review by jeyvison — CHANGES_REQUESTED** [review]
Hi @marcosapmf , Please, see my changes

**Comment by marcosapmf** [comment]
ci:forward

**Comment by marcosapmf** [comment]
Talked to @jeyvison on slack and we agreed that this pull is ready to go

**Pull request closed without merging**